data/saas/config/alchemer_config.yml

```diff
@@ -0,0 +1,46 @@
+saas_config:
+  fides_key: <instance_fides_key>
+  name: Alchemer
+  type: alchemer
+  description: A sample schema representing the Alchemer erasure only integration for Fides
+  user_guide: https://docs.ethyca.com/user-guides/integrations/saas-integrations/alchemer
+  version: 0.1.0
+
+  connector_params:
+    - name: domain
+      label: Domain
+      description: The API domain for Alchemer. Default api.alchemer.com
+      default_value: api.alchemer.com
+    - name: api_token
+      label: API key
+      description: The API key for Alchemer
+      sensitive: True
+    - name: api_token_secret
+      label: API key secret
+      description: The API key secret for Alchemer
+      sensitive: True
```

Comment:
If Alchemer uses the terms `API key` and `API key secret` we should make sure the names match, so `api_key` and `api_token_secret`. It's a bit confusing to use different names for the label and name

Reply:
Just to note - Alchemer's documentation shows them as we would want, api_key and api_key_secret, but the API itself requires that the headers be api_token and api_token_secret which confused me a bit on which way to go with the naming but thanks to Adrian's patience we got it sorted =)